lib/vsc/install/shared_setup.py

```diff
@@ -1357,6 +1357,7 @@ def action_target(target, setupfn=setup, extra_sdist=[], urltemplate=None):
             'setuptools'
         ],
         'excluded_pkgs_rpm': [], # vsc-install ships vsc package (the vsc package is removed by default)
+        'namespace_packages': ['vsc'],
```

Comment:
From http://peak.telecommunity.com/DevCenter/setuptools#namespace-packages:

```
When you declare a package to be a namespace package, it means that the package has no meaningful contents
in its __init__.py, and that it is merely a container for modules and subpackages.
```

So, both modules `fancylogger` (which is just a stub, but fine) and subpackages are allowed in namespaces.

However, I do support getting rid of `vsc.fancylogger`.